NDTensors/test/NDTensorsTestUtils/device_list.jl

```diff
@@ -1,5 +1,5 @@
 using NDTensors: NDTensors
-using Pkg: Pkg
+using Pkg: Pkg, PRESERVE_DIRECT
```

Comment:
Seems like `PRESERVE_DIRECT` isn't needed anymore?

Reply:
Thanks I removed this!